### PR TITLE
Update go modules to include latest k8s.io/kubernetes module on v-1.19

### PR DESCRIPTION
**Pull request by nicolehanjing**
update go modules to include latest k8s.io/kubernetes module on release-1.19 branch
```
$ go get -u k8s.io/kubernetes
go: k8s.io/kubernetes upgrade => v1.19.3
```

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
patched v1.19 for v2 Instance https://github.com/kubernetes/kubernetes/pull/95452 and update go.mod to include the latest version

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by andrewsykim** [comment]
@nicolehanjing can you link the PR with the bug fix we're trying to pull in ?

**Comment by nicolehanjing** [comment]
> @nicolehanjing can you link the PR with the bug fix we're trying to pull in ?

sure, updated the PR description

File: go.mod [hunk 1]
```diff
@@ -42,7 +42,7 @@ require (
 	k8s.io/component-base v0.0.0
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.2.0
-	k8s.io/kubernetes v1.19.1
+	k8s.io/kubernetes v1.19.3
```
Comment (andrewsykim) [inline go.mod:45]: This should probably be `v0.0.0-20201023141757-9e8ad8ce9d8a` until we update this to `v1.1.9.4` in a follow-up PR

Comment (nicolehanjing) [reply go.mod]: updated! I also re-verify that the node registration worked after updating :)

File: go.mod [hunk 2]
```diff
@@ -3,31 +3,31 @@ module k8s.io/cloud-provider-aws
...
+	k8s.io/kubectl => k8s.io/kubernetes/staging/src/k8s.io/kubectl v0.0.0-20201023141757-9e8ad8ce9d8a
+	k8s.io/kubelet => k8s.io/kubernetes/staging/src/k8s.io/kubelet v0.0.0-20201023141757-9e8ad8ce9d8a
+	k8s.io/legacy-cloud-providers => k8s.io/kubernetes/staging/src/k8s.io/legacy-cloud-providers v0.0.0-20201023141757-9e8ad8ce9d8a
+	k8s.io/metrics => k8s.io/kubernetes/staging/src/k8s.io/metrics v0.0.0-20201023141757-9e8ad8ce9d8a
+	k8s.io/sample-apiserver => k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20201023141757-9e8ad8ce9d8a
 )
 
 require (
-	github.com/aws/aws-sdk-go v1.28.2
+	github.com/aws/aws-sdk-go v1.35.5
```
Comment (andrewsykim) [inline go.mod:30]: Do we need to bump aws-sdk-go in this PR? If we can keep v1.28.2 that would be ideal since k8s.io/kubernetes v1.19 is still on v1.28.2

**Review by andrewsykim — COMMENTED** [review]
Comment on bumping AWS SDK, if we can get that back to v1.28.2, then lgtm!

File: go.mod (hunk 2 above)
Comment (nicolehanjing) [reply go.mod]: gotcha, updated!

**Review by andrewsykim — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**